src/client/eppo-client.ts

```diff
@@ -154,6 +163,28 @@ export default class EppoClient {
     this.isObfuscated = isObfuscated;
   }
 
+  async withOverrides(overridePayload: string | undefined): Promise<EppoClient> {
+    if (overridePayload) {
```

Comment:
it's weird to call this with a falsy value and do nothing. I'd say that should be an error, but up to you.

Reply:
It's just a DX thing. 

The main usage will be the something like the following in a customer's API endpoint handler:

`const eppo = await getInstance().withOverrides(req.cookies['eppo-overrides']);`

If the Chrome extension is undefined, then no overrides would be sent. If we make the overridePayload required, we're forcing the customer to do this:

```
const eppo = req.cookies['eppo-overrides'] ? await getInstance().withOverrides(req.cookies['eppo-overrides']) : getInstance()
```

Reply:
makes sense, as long as it's clearly stated in the public docs, the rationale is solid

Reply:
In our own usage, most places we use flags don't have access to the request

Reply:
> In our own usage, most places we use flags don't have access to the request

Yeah, that's the downside to this approach, and it might be a deal breaker. They'd have to manually pass down the eppo instance.